sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/SASTest.groovy

```diff
@@ -669,9 +610,9 @@ class SASTest extends APISpec {
         e.getMessage().contains(parameter)
 
         where:
-        version | creds       || parameter
-        null    | primaryCreds | "version"
-        "v"     | null         | "sharedKeyCredentials"
+        version | creds             || parameter
```

Comment:
I guess the table format of `||` is typo right here. Can you make a singe `|`?

Reply:
Spock allows for `||` in the data table, I don't think it causes any differences in the test run but its mean to be a visual indicator of what the test is compared to in the end.